test/error_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+describe Lotus::Controller::Error do
+  it 'inherits from ::StandardError' do
+    Lotus::Controller::Error.superclass.must_equal StandardError
```

Comment:
@karimmtarek It lacks of test case for `Lotus::Controller::UnknownFormatError` :wink: